pkg/storage/tsdb/caching_bucket.go

```diff
@@ -140,7 +140,7 @@ type MetadataCacheConfig struct {
 }
 
 func (cfg *MetadataCacheConfig) RegisterFlagsWithPrefix(f *flag.FlagSet, prefix string) {
-	f.StringVar(&cfg.Backend, prefix+"backend", "", fmt.Sprintf("Backend for metadata cache, if not empty. Supported values: %s.", CacheBackendMemcached))
+	f.StringVar(&cfg.Backend, prefix+"backend", "", fmt.Sprintf("Backend for metadata cache, if not empty. Supported values: %s, %s, and '' (disable).", CacheBackendMemcached, CacheBackendRedis))
```

Comment:
Can we just use an array? Then the flag description can just reference that array. It is easier to maintain IMO